core/connector/src/main/java/org/exoplatform/ecm/connector/dlp/RestoreFileDLPListener.java

```diff
@@ -1,23 +1,23 @@
 package org.exoplatform.ecm.connector.dlp;
 
 import org.exoplatform.commons.dlp.connector.DlpServiceConnector;
+import org.exoplatform.commons.dlp.domain.DlpPositiveItemEntity;
 import org.exoplatform.commons.dlp.processor.DlpOperationProcessor;
 import org.exoplatform.commons.dlp.service.DlpPositiveItemService;
 import org.exoplatform.commons.utils.CommonsUtils;
 import org.exoplatform.services.listener.Event;
 import org.exoplatform.services.listener.Listener;
 
-public class RestoreFileDLPListener extends Listener<DlpPositiveItemService, String> {
+public class RestoreFileDLPListener extends Listener<DlpPositiveItemService, Object> {
```

Comment:
```suggestion
public class RestoreFileDLPListener extends Listener<DlpPositiveItemService, DlpPositiveItemEntity> {
```
I think it is better to use the object directly instead of casting it ?
WDYT ?

Reply:
Yes , better !